src/chapters/dictionaries/pirate-translator.rst

```diff
@@ -0,0 +1,269 @@
+Code Like a Pirate!
+===================
```

Comment:
There is actually no connection, other than the pirate theme.  Do you think we should resolve this with a note or by changing the accumulator example?

Reply:
I think maybe an acknowledgement at the beginning of this page would be helpful. Something like, "We saw how we could use a dictionary to store words in pirate speak and the English meanings. Let's use dictionaries to make a full translator!" Since both are in pirate speak and dealing with translating pirate speak, drawing attention to it might help students get going with the exercises!

Reply:
Updated the intro to the section to draw attention to the former example and motivate the new task.